gfx/NeoDays/pngs_tiles_10x10/vehicle/vp_door_sliding_0.json

```diff
@@ -0,0 +1 @@
+{"id": "vp_door_sliding_0", "fg": ["3855_vp_door_sliding_0"], "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": ["2286_vp_door_trunk_1"], "bg": []}, {"id": "open", "fg": ["3855_vp_door_sliding_1"], "bg": []}], "bg": []}
```

Comment:
```suggestion
{"id": "vp_door_sliding", "fg": ["3855_vp_door_sliding_0"], "rotates": true, "multitile": true, "additional_tiles": [{"id": "broken", "fg": ["2286_vp_door_trunk_1"], "bg": []}, {"id": "open", "fg": ["3855_vp_door_sliding_1"], "bg": []}], "bg": []}
```

Reply:
Ahh must have misplaced that "_0". and yes, i forgot to change the seat json to use my sprite